book/website/community-handbook/coworking/coworking-organisation.md

```diff
@@ -21,21 +22,21 @@ A few other scenarios could be:
   - If you are working on multiple projects and struggle to focus on one task at a time, coworking can help you put aside a few hours of your time to work with your colleagues who you share the project with.
   - If there is a task you are struggling to complete, you can invite support from your trusted colleagues through coworking calls.
 
-**Target audience**
+### Target audience
 
 If you already know the purpose of your coworking calls, you will be able to identify who your target audience is.
 
 For example, if you host your coworking calls for an open source project such as _The Turing Way_, you can expect your current members, past contributors or even new members to the project to participate.
 If you are part of a small project, you would expect the same familiar faces in all your coworking calls.
 
-**Organisers and hosts**
+### Organisers and hosts
 
 Many tasks go into organising a coworking call (discussed later).
 These tasks can be taken care of by one person like a community manager, project manager, or a volunteer organiser.
 However, it is always a good idea to involve more people as co-organisers so that the coworking call still happens even when the main organiser can't join for any reason.
 To avoid one person to burden themselves with "not so exciting" organisation issues, you can also decide to rotate the role within the members of your project.
 
-**Size of the call**
+### Size of the call
```

Comment:
Maybe there is another section that can be added before or after this one about the "Length of the call"?